RecoMuon/Configuration/python/MergeDisplacedTrackCollections_cff.py

```diff
@@ -46,3 +46,8 @@
     duplicateDisplacedTrackClassifier*
     displacedTracks
     )
+
+# This customization will be removed once we get the templates for
+# phase1 pixel
```

Comment:
@kpedro88 You may want to update the comment :) We already have pixel templates in phase1 (and tracking uses them).